CMake version we're using is too old for LLVM:

```
CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
  CMake 3.20.0 or higher is required.  You are running version 3.19.6
```

--- a/clang/lib/Frontend/CachedDiagnostics.cpp
+++ b/clang/lib/Frontend/CachedDiagnostics.cpp
@@ -25,6 +25,7 @@
 #include "llvm/Support/EndianStream.h"
 #include "llvm/Support/StringSaver.h"
 #include "llvm/Support/YAMLTraits.h"
+#include <variant>
 
 using namespace clang;
 using namespace clang::cas;

--- a/clang/lib/Frontend/CachedDiagnostics.h
+++ b/clang/lib/Frontend/CachedDiagnostics.h
@@ -11,6 +11,8 @@
 
 #include "clang/Basic/LLVM.h"
 #include "llvm/Support/PrefixMapper.h"
+#include <memory>
+#include <optional>
 
 namespace clang {
 class DiagnosticConsumer;

--- a/llvm/include/llvm/TargetParser/SubtargetFeature.h
+++ b/llvm/include/llvm/TargetParser/SubtargetFeature.h
@@ -20,6 +20,7 @@
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/ADT/bit.h"
 #include "llvm/Support/MathExtras.h"
 #include <array>
 #include <initializer_list>

